### PR TITLE
First column in sheet now has string value

### PR DESCRIPTION
**Pull request by johnplacek**

### CODE REVIEW

File: ekglib/xlsx_parser/parser.py [hunk 1]
```diff
@@ -381,8 +381,10 @@ def parse_row(self, sheet_iri, sheet_name_escaped, df, row):
...
         self.g.add((resource_iri, RAW.isRowInView, sheet_iri))
-        for column_number, cell in enumerate(row[2:]):
-            self.parse_cell(resource_iri, df, row_number, column_number, cell, self.column_names[column_number + 1])
+        row_without_row_number = row[1:]
+        for column_number, cell in enumerate(row_without_row_number):
+            column_name = self.column_names[column_number]
+            self.parse_cell(resource_iri, df, row_number, column_number, cell, column_name)
 
     def construct_resource_iri(self, sheet_name_escaped, row_number, legacy_id):
         legacy_id = strip_end(legacy_id, key_column_suffix)
```
Comment (somanythings) [inline ekglib/xlsx_parser/parser.py:390]: a neat idiom for this is 
```v[:v.rfind("_legacy")]```

so 
```legacy_id[:legacy_id.rfind(key_column_suffix)]```

Comment (somanythings) [reply ekglib/xlsx_parser/parser.py:390]: pythonistas LOVE the stringslice

File: tests/test_xlsx.py [hunk 2]
```diff
@@ -47,6 +45,42 @@ def test_parse_column_name(self):
...
+            data_source_code="def",
+            key_column_number=1,
+            strip_any_prefix=False,
+            output=None
+        )
+        parser = ekglib.XlsxParser(args)
+
+        actual = set(parser.g.triples((None,
+                                       RDF.type,
+                                       term.URIRef("https://ekgf.org/ontology/raw/StringValue"))))
```
Comment (somanythings) [inline tests/test_xlsx.py:65]: RDFStringValue  on line 65, too

File: tests/test_xlsx.py [hunk 3]
```diff
@@ -47,6 +45,42 @@ def test_parse_column_name(self):
...
+        RDFStringValue = term.URIRef(
+            "https://ekgf.org/ontology/raw/StringValue")
+        self.assertSetEqual(actual,
+                            {
+                                (term.URIRef("https://kg.your-company.kom/id/data-name-popular-value-1"),
```
Comment (somanythings) [inline tests/test_xlsx.py:70]: we should probably lift out the "https://kg.your-company.kom/id" to kgiri
i wouldn't be surprised if there's magic in rdflib for that that deals with cases I don't know about that a string + won't.

Comment (somanythings) [reply tests/test_xlsx.py:70]: maybe 
```python
def kgiri(path):
   term.URIRef("https://kg.your-company.kom/id/" + path)
```

File: tests/test_xlsx.py (hunk 2 above)
Comment (johnplacek) [reply tests/test_xlsx.py:65]: Done

**Pull request merged**